### DISCUSSION TITLE
Can more than one answer be marked as the answer in a Q&A?

### DISCUSSION THREAD

**Discussion by jherskow** [drknzz/GitHub-Achievements#5] [Q&A] [answered]
Curious as to how this would work in terms of the galaxy brain achievement

**Comment by drknzz** [comment]
You could try marking multiple answers

**Reply by drknzz** [reply]
Yeah, unfortunately there can be only one answer to a discussion at a time.
Additionally you cannot get the achievement by marking your answer yourself.

**Reply by jherskow** [reply]
cool
im gonna unmark this answer and see if you achievement sticks

**Reply by jherskow** [reply]
Looks like it worked!!

**Reply by umutguden** [reply]
Yey!

**Reply by jasonlittleton24** [reply]
Amazing!

**Reply by 0x5h31d0n** [reply]
Hey

**Reply by Mx444** [reply]
+1

**Reply by Suhaibusman** [reply]
+1

**Reply by christivn** [reply]
+1

**Reply by 0x5t4l1n** [reply]
+1

**Reply by 0x5t4l1n** [reply]
hii

**Reply by phamduyaaaa** [reply]
+1

**Reply by a-ramirezzz** [reply]
+1

**Reply by PYastreboff** [reply]
+1

**Reply by joebertcerezo** [reply]
+1

**Reply by Sopheaktraleng** [reply]
+1

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Reply by este0207** [reply]
+1

**Reply by Juan-Miguel-alvarado** [reply]
+ 1

**Reply by Faker-DJ** [reply]
+1

**Reply by chuck-bear** [reply]
+1

**Comment by drknzz** [comment]
🤔

**Reply by jherskow** [reply]
Seems you need to unmark one to get the other

BUT

It seems like you kept your galaxy brain after i unmarked your answers!

So maybe it stays?

great meta either way

**Reply by japanihon** [reply]
Sounds like the system works well !

**Reply by 0x5t4l1n** [reply]
hwlloq

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by Abeir01** [comment]
You could try marking multiple answers

**Reply by japanihon** [reply]
Ok, hope works for me

**Reply by 0x5t4l1n** [reply]
man

**Reply by 0x5t4l1n** [reply]
+1

**Reply by Juan-Miguel-alvarado** [reply]
1

**Reply by loofiboss-bit** [reply]
1

**Comment by japanihon** [comment]
You could try marking multiple answers

**Reply by drknzz** [reply]
Hey, I marked this answer, but in order for you to get the achievement you need to have 2 marked answers. So if you just commented on any other discussion in this repository, I'll mark it and you'll get the badge :)

**Reply by japanihon** [reply]
Hi Kamil, I already do it, thanks a lot ! Have a great code day :)

**Reply by 0x5t4l1n** [reply]
ok goof :)

**Reply by PYastreboff** [reply]
+1

**Reply by Juan-Miguel-alvarado** [reply]
1

**Reply by chuck-bear** [reply]
+1

**Comment by japanihon** [comment]
Sounds like the system works well ! :)

**Reply by drknzz** [reply]
One more thing I forgot to mention😅
I've already marked one of your answers here but I cannot mark 2 answers in the same discussion at the same time :/
So to get the achievement just answer another discussion please 😉

You can answer this discussion for example:
https://github.com/drknzz/GitHub-Achievements/discussions/19

**Reply by japanihon** [reply]
😀 Thanks !

**Reply by 0x5t4l1n** [reply]
ooh nice

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by umutguden** [comment]
Just testing.

**Reply by drknzz** [reply]
If you want the Galaxy Brain achievement, answer another discussion, so I can mark it 😉

**Reply by umutguden** [reply]
> If you want the Galaxy Brain achievement, answer another discussion, so I can mark it 😉

I'm gonna do it!

**Reply by MatanZada** [reply]
me to

**Reply by vinuwa98** [reply]
Just testing.

**Reply by jvcByte** [reply]
> If you want the Galaxy Brain achievement, answer another discussion, so I can mark it 😉

up and grateful

**Reply by Mx444** [reply]
+1

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by andrewtanjaya** [comment]
yes u can, try mark mine :)

**Reply by vinuwa98** [reply]
Just testing.

**Reply by 0x5t4l1n** [reply]
good

**Comment by MatanZada** [comment, marked as answer]
Just testing.

**Reply by vinuwa98** [reply]
Just testing.

**Reply by 0x5t4l1n** [reply]
nice

**Reply by PYastreboff** [reply]
+1

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by hamimul** [comment]
How does marking one's answer affect the achievement, and what happens after marking it?

**Reply by midhxn** [reply]
You have to answer 2 discussions to achieve the Galaxy Brain Badge. **Marking one's reply as answer gives them one step ahead to the achievement.** After marking, it is considered as the answer of the question

**Reply by vinuwa98** [reply]
test 1

**Reply by 0x5t4l1n** [reply]
How does marking one's answer affect the achievement, and what happens after marking it?

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by zMocao** [comment]
Hmm

**Reply by 0x5t4l1n** [reply]
ok

**Comment by zMocao** [comment]
Mark me

**Reply by vinuwa98** [reply]
Just testing.

**Reply by 0x5t4l1n** [reply]
nice

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by ogprzemo** [comment]
test

**Reply by vinuwa98** [reply]
Just testing

**Reply by malikrohail** [reply]
test

**Comment by lucasfiduniv** [comment]
teste

**Comment by zaurh** [comment]
dzien dobry

**Comment by elmouaze** [comment]
test

**Comment by ramadhanaha95** [comment]
Just testing

**Comment by Mx444** [comment]
test

**Comment by Mx444** [comment]
test

**Comment by RaghavRamteke** [comment]
test

**Comment by a-ramirezzz** [comment]
Just testing

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by maiz-an** [comment]
### if you visit my site will be index on google so, please
https://www.mzieos.com/

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by 0bvim** [comment]
@MatanZada is very good nickname... Exists a Brazilian band that's called `matanza`.

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by ashfakhm** [comment]
test

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by milentijev1c** [comment]
Just wanted to ask are there any info on the 2 badges that doesn't have an explanation of how to get them?

**Reply by William0209** [reply]
Heart On Your Sleeve is prob just liking and showing love on the platform, and Open Sourcerer, I would guess could be acquired through contributing to open-source projects. Just my guesses, though!

**Reply by Juan-Miguel-alvarado** [reply]
1

**Comment by spidybroz** [comment]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Comment by este0207** [comment]
test

**Reply by brianhexer** [reply]
test

**Comment by tasnimultas67** [comment]
testing purpose, thanks advance

**Reply by AteebNoOne** [reply]
Test back

**Reply by Juan-Miguel-alvarado** [reply]
2

**Comment by AteebNoOne** [comment]
Testing purpose, thanks advance (2)

**Reply by Juan-Miguel-alvarado** [reply]
2

**Comment by Juan-Miguel-alvarado** [comment]
mark me pls?

**Comment by Sopheaktraleng** [comment]
just testing

**Reply by Juan-Miguel-alvarado** [reply]
2

**Comment by Juan-Miguel-alvarado** [comment]
https://github.com/Juan-Miguel-alvarado/deepsearch
give stars Pls!

**Comment by chuck-bear** [comment]
just testing

**Reply by Juan-Miguel-alvarado** [reply]
=1
